### PR TITLE
KSQL-365  Server doesn't show any sign of life at startup

### PR DESCRIPTION
**Pull request by big-andy-coates**
…, even when it is alive

- Moved welcome message generation into common location
- Enhanced rest app to output welcome message, if a console is attached.
- Updated copyright year.

## New Server output

**NOTE**: This is only output if there is a console attached to the process.
**NOTE**: It assumes a 80 char wide console, unlike the CLI output which adjusts to the console width.

```
                    ======================================
                     =      _  __ _____  ____  _          =
                     =     | |/ // ____|/ __ \| |         =
                     =     | ' /| (___ | |  | | |         =
                     =     |  <  \___ \| |  | | |         =
                     =     | . \ ____) | |__| | |____     =
                     =     |_|\_\_____/ \___\_\______|    =
                     =                                    =
                     =   Streaming SQL Engine for Kafka   =
                     ======================================

Copyright 2018 Confluent Inc.

To access the KSQL CLI, run:
ksql-cli remote http://localhost:8090

Tp access the UI, point your browser at:
http://localhost:8090/index.html
```

## New CLI welcome message:

```
                     ======================================
                     =      _  __ _____  ____  _          =
                     =     | |/ // ____|/ __ \| |         =
                     =     | ' /| (___ | |  | | |         =
                     =     |  <  \___ \| |  | | |         =
                     =     | . \ ____) | |__| | |____     =
                     =     |_|\_\_____/ \___\_\______|    =
                     =                                    =
                     =   Streaming SQL Engine for Kafka   =
                     ======================================

Copyright 2018 Confluent Inc.

CLI v0.5, Server v<unknown> located at http://localhost:8090

Having trouble? Type 'help' (case-insensitive) for a rundown of how things work!

ksql> 
```

### CODE REVIEW

File: ksql-common/src/test/java/io/confluent/ksql/util/WelcomeMsgUtilsTest.java [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        + "                     =   Streaming SQL Engine for Kafka   =\n"
+        + "                     ======================================\n"
+        + "\n"
+        + "Copyright 2018 Confluent Inc.\n"
+        + "\n")
+    );
+  }
+
+  @Test
+  public void shouldOutputShortWelcomeMessageIfConsoleIsWideEnough() {
```
Comment (dguy) [inline ksql-common/src/test/java/io/confluent/ksql/util/WelcomeMsgUtilsTest.java]: `...IfConsoleNotWideEnough"?

File: ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java [hunk 2]
```diff
@@ -405,6 +412,36 @@ static void createCommandTopicIfNecessary(final KsqlRestConfig restConfig,
...
+          .replace("0.0.0.0", "localhost");
+
+      writer.printf("Server %s listening on %s%n", version, listener);
+      writer.println();
+      writer.println("To access the KSQL CLI, run:");
+      writer.println("ksql-cli remote " + listener);
+      writer.println();
+
+      if (isUiEnabled) {
+        writer.println("Tp access the UI, point your browser at:");
```
Comment (dguy) [inline ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java]: typo: Tp -> To

**Comment by rmoff** [comment]
https://github.com/confluentinc/ksql/issues/797 raised for the ASCII art idea. 

I think marketing should comment on the wording - I'll post a note on Slack.

**Comment by big-andy-coates** [comment]
@rmoff Perfect. Thanks Robin.

**Comment by blueedgenick** [comment]
yes please for 'Apache Kafka R'  (note: not 'TM' anymore!)

agreed that the 'confluent' part is redundant.

**Comment by big-andy-coates** [comment]
Product requirements encorporated. New common output looks like:

{noformat}
                   
                   ==========================================
                   =       _  __ _____  ____  _             =
                   =      | |/ // ____|/ __ \| |            =
                   =      | ' /| (___ | |  | | |            =
                   =      |  <  \___ \| |  | | |            =
                   =      | . \ ____) | |__| | |____        =
                   =      |_|\_\_____/ \___\_\______|       =
                   =                                        =
                   = Streaming SQL Engine for Apache Kafka® =
                   ==========================================

Copyright 2018 Confluent Inc.

{noformat}

**Comment by rmoff** [comment]
⭐️ 👍 thanks @big-andy-coates!

**Pull request merged**